analytics/anomaly.md

```diff
@@ -0,0 +1,30 @@
+# Anomaly Detection
+
+With algorithm "Seasonal Hybrid ESD (S-H-ESD)", created by Twitter, it detects anomaly in the presence of seasonality and an underlying trend
+
+## Input Data
+Input data should be a time series data. Each row should represent one observation (e.g. one row per date/time). It should have following columns.
```

Comment:
"One row per date/time" is not really true. We can handle multiple rows with a same time/date by aggregation.
We can just say, `Each row should represent one observation with date/time.`